sql/hive/src/test/scala/org/apache/spark/sql/hive/MultiDatabaseSuite.scala

```diff
@@ -80,7 +80,7 @@ class MultiDatabaseSuite extends QueryTest with SQLTestUtils with TestHiveSingle
               |CREATE TABLE t1
               |USING parquet
               |OPTIONS (
-              |  path '$path'
+              |  path '${dir.toURI.toString}'
```

Comment:
I see, let me correct it for the former. In case of `path`, it is being used above. I thought this is a minimised change because this is the only problematic line, parsing the path wrongly on Windows. For example, `C:\tmp\a\b\c` becomes `C:    mpabc`.